resolwe/flow/managers/listener.py

```diff
@@ -509,6 +509,16 @@ def handle_download_finished(self, obj):
             storage_location.files.add(*default_storage_location.files.all())
             storage_location.status = StorageLocation.STATUS_DONE
             storage_location.save()
+            # Add lock to it since it is required for processing. Do not change
+            # the reason since it is used to unlock storage locations after
```

Comment:
"since it is used to unlock" - is this true or is it unlocked based on the cause?